src/components/AdminRegisterPage/index.jsx

```diff
@@ -40,7 +41,16 @@ const AdminRegisterPage = () => {
         logError(error);
       }
     };
-    getEnterpriseBySlug();
+    LmsApiService.loginRefresh().then(_ => {
+      const obfuscatedId = uuidv5(String(_.userId), uuidv5.DNS);
```

Comment:
Hm, i'm not sure this is appropriate use of the `_` variable name.  Perhaps `data` will do?
```suggestion
      const obfuscatedId = uuidv5(String(data.userId), uuidv5.DNS);
```